### DISCUSSION TITLE
wagmi 1.0 support

### DISCUSSION THREAD

**Discussion by Garfield550** [family/connectkit#219] [Ideas]
wagmi v1.0 dropped ethers.js support and use its own library called [viem](https://viem.sh).

**Comment by benjitaylor** [comment]
We'll be rolling this out shortly @Garfield550. 🤝

**Reply by richardryangarcia** [reply]
love to hear it ! 🎉

**Comment by lochie** [comment]
This is now live in ConnectKit `1.4.0`
